.yamato/input-system-mobile-functional-tests.yml

```diff
@@ -59,7 +59,7 @@ inputsystem-mobilefunctionaltests_-_2022_3_-_ios:
     image: package-ci/macos-13:default
     type: Unity::mobile::iPhone
     flavor: m1.mac
-    model: SE
+    model: SE-Gen3
```

Comment:
Regeneration CI should restore it to `model: SE`. `model: SE-Gen3` is for 6000.3+ versions.